app/views/talents/edit.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="container">
+  
+  <div class="row mt-20 mb-20">
+    <div class="col-xs-6 col-sm-6 col-md-6 col-lg-6">
+      <h2>Edit Talent</h2>
+      <!-- Button trigger modal -->
+    </div>
+  </div>
+  <hr>
+  <%= render partial: 'form', locals: { :talent => @talent } %>
```

Comment:
`partial` & `locals` keywords are no longer needed, you can write the above like this: 

```
render 'form', talent: @talent
```